sql_generators/mobile_kpi_support_metrics/templates/retention_clients.view.sql

```diff
@@ -108,11 +91,8 @@ INNER JOIN
   ON clients_daily.submission_date = active_users.retention_seen.day_27.metric_date
   AND clients_daily.client_id = active_users.client_id
   AND clients_daily.normalized_channel = active_users.normalized_channel
-  {% if attribution_fields %}
-    LEFT JOIN
-      attribution
-      ON clients_daily.client_id = attribution.client_id
-      AND clients_daily.normalized_channel = attribution.normalized_channel
-  {% endif %}
+LEFT JOIN
```

Comment:
Conditional now longer needed as now all of the product will have an attribution table generated regardless of whether the product telemetry data has any attribution info.